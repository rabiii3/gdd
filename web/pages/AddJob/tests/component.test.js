import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore  from '../../../store/configureStore.test';
import { AddJobDescription } from '..';

describe( 'app | pages | AddJobDescription | component', () => {
  it('should match snapshot', () => {
    const state = {};
    const props = {
      classes:{},
    };
    const store = configureStore(state);
      const wrapper = shallow(
        <Provider store={store}>
          <AddJobDescription {...props} />
        </Provider>,
      );
      expect(wrapper).toMatchSnapshot();
  });
});