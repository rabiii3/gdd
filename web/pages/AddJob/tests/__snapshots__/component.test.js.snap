// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app | pages | AddJobDescription | component should match snapshot 1`] = `
<Component
  classes={Object {}}
/>
`;
